### PR TITLE
feat: rename :global: env to "default"

### PR DESCRIPTION
**Pull request by ivarconr**
Our testing and internal validation has proven that
the :global: environment concept confuses people more
than the problems it solves. We have thus decided to
group all configuration that was created before the
environment concept was introdcued in to the "default
environment. This would still make everything work
as before in addition to introducing the env concept.

### CODE REVIEW

File: src/migrations/20210920104218-rename-global-env-to-default-env.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict';
```
[the remark this answers is not in the document]
Comment (ivarconr) [reply src/migrations/20210920104218-rename-global-env-to-default-env.js:1]: help

[the remark this answers is not in the document]
Comment (ivarconr) [reply src/migrations/20210920104218-rename-global-env-to-default-env.js:1]: @sonatype-lift help

[the remark this answers is not in the document]
Comment (ivarconr) [reply src/migrations/20210920104218-rename-global-env-to-default-env.js:1]: @sonatype-lift ignore

File: src/migrations/20210920104218-rename-global-env-to-default-env.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        UPDATE feature_strategies SET environment = ':global:' WHERE environment = 'default';
+        UPDATE feature_environments SET environment = ':global:' WHERE environment = 'default';
+        UPDATE project_environments SET environment_name = ':global:' WHERE environment_name = 'default';
+        DELETE FROM environments WHERE name = 'default';
+        `,
+        cb,
+    );
+};
+
+module.exports = { up, down };
```
[the remark this answers is not in the document]
Comment (chriswk) [reply src/migrations/20210920104218-rename-global-env-to-default-env.js:33]: @sonatype-lift ignore

**Pull request merged**